applications/source/debezium-source/README.adoc

```diff
@@ -0,0 +1,343 @@
+
+== Options
+
+//tag::configuration-properties[]
+$$debezium.copy-headers$$:: $$Copy Change Event headers into Message headers.$$ *($$Boolean$$, default: `$$true$$`)*
+$$debezium.debezium-native-configuration$$:: $$<documentation missing>$$ *($$Properties$$, default: `$$<none>$$`)*
+$$debezium.format$$:: $$Change Event message content format. Defaults to 'JSON'.$$ *($$DebeziumFormat$$, default: `$$<none>$$`, possible values: `JSON`,`AVRO`,`PROTOBUF`)*
+$$debezium.offset-commit-policy$$:: $$The policy that defines when the offsets should be committed to offset storage.$$ *($$DebeziumOffsetCommitPolicy$$, default: `$$<none>$$`, possible values: `ALWAYS`,`PERIODIC`,`DEFAULT`)*
+$$debezium.properties$$:: $$Spring pass-trough wrapper for debezium configuration properties. All properties with a 'debezium.properties.*' prefix are native Debezium properties.$$ *($$Map<String, String>$$, default: `$$<none>$$`)*
+//end::configuration-properties[]
+
+==== Event flattening configuration
+
+Debezium provides a comprehensive message format, that accurately details information about changes that happen in the system.
+Sometime this format, though,  might not be suitable for the downstream consumers, that might require messages that are formatted so that field names and values are presented in a simplified, `flattened` structure.
+
+To simplify the format of the event records that the Debezium connectors produce, you can use the https://debezium.io/documentation/reference/stable/transformations/event-flattening.html[Debezium event flattening] message transformation.
+Using the https://debezium.io/documentation/reference/stable/transformations/event-flattening.html#_configuration[flattering configuration] you can configure simple messages format like this:
+
+[source, bash]
+----
+--debezium.properties.transforms=unwrap
+--debezium.properties.transforms.unwrap.type=io.debezium.transforms.ExtractNewRecordState
+--debezium.properties.transforms.unwrap.drop.tombstones=false
+--debezium.properties.transforms.unwrap.delete.handling.mode=rewrite
+--debezium.properties.transforms.unwrap.add.fields=name,db
+----
+
+==== Connectors properties
+
+The table below lists all available Debezium properties for each connecter.
+Those properties can be used by prefixing them by the `debezium.properties.` prefix.
+
+.Table of the configuration properties for every Debezium connector.
+|===
+| Connector | Connector properties
+
+|https://debezium.io/documentation/reference/2.2/connectors/mysql.html[MySQL]
```

Comment:
Not sure what you mean? the https://debezium.io/documentation/reference/2.2/connectors/mysql.html is opening ok and corresponds to the connector version we support.

Reply:
Right. I just want to avoid burden of supporting updates to this doc when we just would move to Debezium `2.3`, for example.
Why just not have a "versionless" link in the doc while on upgrade we would mostly just change the version in the POM?